rpcs3/Emu/Cell/lv2/sys_vm.h

```diff
@@ -2,6 +2,7 @@
 
 #include "Emu/Memory/vm_ptr.h"
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/IdManager.h"
```

Comment:
I wouldn't put this in the header.

Reply:
Including that header in `sys_vm.h` wasn't my decision, it was Nekotekina's back in 2016 (c95f6c8c560aac219babbb62b48fc9b402e76ac5). This commit is supposed to be a clean revert of 4599d5841353fdf8096191c30a592fedc844efce, which happened to move that `#include`.
Amending the revert feels wrong to me, so would you like me to add another commit with the following changes?
- Remove `ErrorCodes.h` and `IdManager.h` in the header, and `vm_locking.h` in `sys_vm.cpp`
- Forward-declare `struct lv2_memory_container` in the header instead of `#include "sys_memory.h"`
- Add `ErrorCodes.h`, `IdManager.h`, `sys_memory.h`, `CPUThread.h`, and `vm.h` to `sys_vm.cpp`

Reply:
Who cares, if you add a commit it will just get squashed on merge anyways.

Reply:
Also just modify this one include line, dont touch the others please.